frontend/apps/vft-extended/src/pages/home/styles.ts

```diff
@@ -0,0 +1,47 @@
+import type { CSSProperties } from 'react';
+
+export const styles: { [key: string]: CSSProperties } = {
```

Comment:
I would do styling via `Home.module.scss`, inline CSS feels wrong here.

Also, why not to use components from our UI kits?